File: src/BusinessTemplates/CSModernBusinessApp/OpenSilverBusinessApplication.Server/Data/ApplicationDbContext.cs
using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore;
using $ext_safeprojectname$.Models;

public class ApplicationDbContext : IdentityDbContext<User>
{
    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
        : base(options)
    {
    }
}

File: src/BusinessTemplates/VBModernBusinessApp/OpenSilverBusinessApplication.Server/Data/ApplicationDbContext.vb
Imports Microsoft.AspNetCore.Identity.EntityFrameworkCore
Imports Microsoft.EntityFrameworkCore
Imports $ext_safeprojectname$.Models

Public Class ApplicationDbContext
    Inherits IdentityDbContext(Of User)

    Public Sub New(options As DbContextOptions(Of ApplicationDbContext))
        MyBase.New(options)
    End Sub

End Class